standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -133,6 +133,10 @@ public InProcPravegaCluster build() {
             //For SegmentStore
             Preconditions.checkState(isInProcSegmentStore || this.segmentStorePorts != null, "SegmentStore ports not declared");
 
+            //Check TLS related parameters
+            Preconditions.checkState(!enableTls || !Strings.isNullOrEmpty(this.keyFile) || !Strings.isNullOrEmpty(this.certFile),
```

Comment:
The logic seems wrong to me, because `keyFile` must be non-null when `enableTls` is true AFAIK.

Reply:
@arvindkandhare is a keyfile required when TLS is enabled?  Is the above check related to that?

Reply:
Yes. A `keyFile` and `certFile` is required only when `TLS` is enabled. The above check is related to that.

Reply:
Fixed.